<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkVisualization</class>
 <widget class="QDialog" name="NetworkVisualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Visualization</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Configuration</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>257</width>
        <height>234</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="chkHideAntiCorrelations">
         <property name="text">
          <string>Hide Anti-Correlations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkScaleHubs">
         <property name="text">
          <string>Scale nodes by hubs size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkScaleEdgesScore">
         <property name="text">
          <string>Scale edges by score</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkCommunitiesColor">
         <property name="text">
          <string>Color nodes by communities</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Layout:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Random Seed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="txtRandomSeed"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbLayout">
           <item>
            <property name="text">
             <string>Components</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hierarchical UD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hierarchical DU</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="cmdReload">
         <property name="text">
          <string>Reload Network</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
